packages/@aws-cdk-testing/framework-integ/test/aws-events/test/integ.eventbus-grants.ts

```diff
@@ -0,0 +1,176 @@
+import { App, Stack, Duration, CfnOutput, Fn } from 'aws-cdk-lib';
+import * as events from 'aws-cdk-lib/aws-events';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import * as tasks from 'aws-cdk-lib/aws-stepfunctions-tasks';
+import * as ssm from 'aws-cdk-lib/aws-ssm';
+import { ExpectedResult, IntegTest, Match } from '@aws-cdk/integ-tests-alpha';
+import * as cxapi from 'aws-cdk-lib/cx-api';
+import { STANDARD_NODEJS_RUNTIME } from '../../config';
+
+// Create a single app for both stacks
+const app = new App();
```

Comment:
I think in this integ test case, there is no difference between with feature flag, and without. In both cases the permissions will be added to the Role .. may be this test case is designed to verify that the current customer code will stay working. so, can we have a new integ test case to verify the new behaviour of grant method, and also, to verify that we can call grant method more than one in the new approach.